### PR TITLE
Claim all page

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
RapidPro - Visually build nationally scalable mobile applications 2021-02-23 21-51-30

The new view similar to the existing claim page just no type is filtered out

File: temba/channels/models.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ class IVRProtocol(Enum):
     code = None
     slug = None
     category = None
+    beta_only = False
```
Comment (norkans7) [inline temba/channels/models.py:57]: the flag for channel that are still beta gated, used by `is_available_beta` method that can be overwritten in any type

defaults to False to make that type available to anyone

Comment (rowanseymour) [reply temba/channels/models.py:57]: So this is to make sure that beta-only channels are still filtered even in the non-filtered view right?

Do we need `is_available_beta` ? Can't we just check `beta_only` in the code that lists channel types?

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: temba/channels/models.py (hunk 1 above)
Comment (norkans7) [reply temba/channels/models.py:57]: Yes we need it to allow overwriting the method to we can disable channel type no longer used such as old facebook

Comment (rowanseymour) [reply temba/channels/models.py:57]: Right so some channels should always be hidden (beta or discontinued) even in no-filter mode. It's just having `beta_only` and `is_available_beta` and `is_available_to` feels more complicated than it needs to be.

What if `is_available_to` returns a tuple of 2 bools - the first is whether the channel should be available in filtered mode, the second if it should be available in non-filtered mode?

Comment (norkans7) [reply temba/channels/models.py:57]: Okay

File: temba/channels/types/facebookapp/type.py [hunk 2]
```diff
@@ -28,7 +28,9 @@ class FacebookAppType(ChannelType):
 
     claim_blurb = _(
         "Add a %(link)s bot to send and receive messages on behalf of one of your Facebook pages for free. You will "
-        "need to connect your page by logging into your Facebook and checking the Facebook page to connect."
+        "need to connect your page by logging into your Facebook and checking the Facebook page to connect. "
```
Comment (norkans7) [inline temba/channels/types/facebookapp/type.py:31]: Clarify more on Facebook page requirements

File: temba/channels/models.py [hunk 3]
```diff
@@ -95,13 +96,16 @@ class IVRProtocol(Enum):
 
     def is_available_to(self, user):
         """
-        Determines whether this channel type is available to the given user, e.g. check timezone
+        Determines whether this channel type is available to the given user considering the region and when not considering region, e.g. check timezone
         """
+        region_ignore_visible = (not self.beta_only) or user.groups.filter(name="Beta")
```
Comment (rowanseymour) [inline temba/channels/models.py:101]: `user.is_beta()`

**Review by rowanseymour — APPROVED** [review]

**Comment by norkans7** [comment]
RapidPro - Visually build nationally scalable mobile applications 2021-02-24 20-39-54

**Review by nicpottier — APPROVED** [review]

**Pull request merged**